schemas/json/aas.json

```diff
@@ -1207,6 +1224,34 @@
         "keys"
       ]
     },
+    "ModelReferenceToSubmodel": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Reference"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "type": {
+              "const": "ModelReference"
```

Comment:
```suggestion
              "const": "Submodel"
```

Reply:
Thanks for the review @BirgitBoss.
The schema that I provided only enforces that the reference should be `ModelReference` and not `ExternalReference` with only one `Key`. So for enforcing that the key should refer to a `Submodel` then, we should further extend it.

In any case the fix should be in the aas-core schema generator.